src/routes.js

```diff
@@ -1,10 +1,12 @@
-import React, { lazy, Suspense } from "react";
-import { Switch, Route, BrowserRouter as Router } from "react-router-dom";
-import Navbar from "./components/Navbar";
-import Footer from "./components/Footer";
+import React, { lazy, Suspense } from 'react';
+import { Switch, Route, BrowserRouter as Router } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Footer from './components/Footer';
 
 const render = Comp => props => <Comp {...props} />;
```

Comment:
what is `render` function used for?

Reply:
It was present in the Boiler plate, and I didn't actually notice it until now.

Reply:
I removed it.